package.json

```diff
@@ -45,8 +46,8 @@
     "webpack-merge": "^4.1.1"
   },
   "scripts": {
-    "dev": "concurrently 'yarn run dev:server' 'yarn run dev:client'",
-    "dev:server": "NODE_ENV=development BABEL_ENV=node babel-watch src/server/index.js -w src -e js,jsx -i dist",
+    "dev": "concurrently \"yarn run dev:server\" \"yarn run dev:client\"",
+    "dev:server": "babel-watch src/server/index.js -w src -e js,jsx -i dist",
```

Comment:
This line is where youll need to actually use cross-env to set the environment variables that were deleted